ee/WFLY-19062-[PREVIEW]-Support_for_Jakarta_MVC_2.1.adoc

```diff
@@ -0,0 +1,175 @@
+* [ ] Experimental
+
+* [X] Preview
+
+* [ ] Community
+
+* [ ] default
+
+=== Dev Contacts
+
+* mailto:brian.stansberry@redhat.com[Brian Stansberry]
+
+=== QE Contacts
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE. 
+// Discuss with QE during the Kickoff state to decide this
+* [X] Engineering
+
+* [ ] QE
+
+=== Affected Projects or Components
+
+* mvc-krazo-feature-pack
+
+=== Other Interested Projects
+
+=== Relevant Installation Types
+// Remove the x next to the relevant field if the feature in question is not relevant
+// to that kind of WildFly installation
+* [x] Traditional standalone server (unzipped or provisioned by Galleon)
+
+* [x] Managed domain
+
+* [x] OpenShift s2i
+
+* [x] Bootable jar
+
+== Requirements
+
+=== Hard Requirements
+
+* Provide an extension/subsystem that makes modules containing the Jakarta MVC 2.1 api 
+  and the Eclipse Krazo implementation available to an application.
+** Only the Jakarta MVC API module will be jboss-api=public; other modules added here will be jboss-api=private
+* Provide an 'mvc-krazo' Galleon layer
+** The layer will depend on the 'bean-validation', 'cdi', 'jaxrs-core' and 'jsf' layers, in line
+   with Jakarta MVC's dependencies on other EE specifications.
+* Provide an extension/subsystem artifact jar, and a 'galleon-shared' src zip for use
+  in other feature packs.
+* Incorporate the above in the 'wildfly' and 'wildfly-preview' feature packs.
+* Extension will advertise itself as having stability level 'preview'.
+* JBoss Modules modules associated with the feature will not be provisioned if the user has disabled provisioning of 'preview' stability features.
+
+=== Nice-to-Have Requirements
+// Requirements in this section do not have to be met to merge the proposed functionality.
+// Note: Nice-to-have requirements that don't end up being implemented as part of
+// the work covered by this proposal should be moved to the 'Future Work' section.
+
+
+=== Non-Requirements
+// Use this section to explicitly discuss things that readers might think are required
+// but which are not required.
+ * Inclusion of the extension/subsystem in any standard OOTB config, including docs/examples configs.
+ * Inclusion in any of the WildFly user boms.
+ * Quickstart.
+
+=== Future Work
+// Use this section to discuss requirements that are not addressed by this proposal
+// but which may be addressed in later proposals.
+* Subsequent promotion to 'community' stability.
+* Possible move into the 'wildfly-ee' feature pack.
+** This would reflect a 'promotion' in terms of WildFly's long term compatibility expectations for Jakarta MVC itself.
+* Add a quickstart
+** Requires promotion to 'community' or Quickstart project support for preview features.
+
+== Backwards Compatibility
+
+// Does this enhancement affect backwards compatibility with previously released
+// versions of WildFly?
```

Comment:
I think the answer is that this enhancement does not affect the backward compatibility with previously released versions of WildFly as this is a new feature.

Reply:
This is answered in the subsections below.